### ISSUE TITLE
[Bug Report] Direct Download Links don't include baseurl

### ISSUE THREAD

**Issue by flying-sausages** [hkalexling/Mango#103]
**Describe the bug**
When a baseurl is set in the config and you want to direct download a file, the opds link doesn't include the baseurl

**To Reproduce**
1. Set basurl
2. log in to manfo
3. open the context card for a book
4. click download link
5. link doesn't contain baseurl

**Expected behavior**
A link with the right baseurl

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04
 - Browser Firefox
 - Mango Version Version 0.12.0

**Docker (if you are running Mango in a Docker container)**
- Nope

**Additional context**
> config.yml
```yml
...
base_url: /mango
library_path: /opt/mango/library
db_path: /opt/mango/.config/mango/mango.db
...
```

**Comment by hkalexling** [comment]
Ah, you are right! Will fix it in the next release 👍

**Comment by hkalexling** [comment]
Fixed in [v0.12.1](https://github.com/hkalexling/Mango/releases/tag/v0.12.1). Thanks for the bug report!
